src/components/Button/index.tsx

```diff
@@ -400,6 +400,7 @@ const FwdButton = ({
         download={download}
         ref={innerRef}
         as={as}
+        role={as === 'a' ? 'button' : undefined}
```

Comment:
Well if it's a link it's not a button 😄 

you can remove the role altogether I think

Reply:
A11y throws an error if the button is a link with href.
the `a` element cant have aria-label

Reply:
so it doesn't need for aria label if its link ?